library/general/src/lib/yast2/popup_rspec.rb

```diff
@@ -0,0 +1,20 @@
+require "yast"
+
+Yast.import "UI"
+
+require "yast2/popup"
+
+def expect_to_show_popup_which_return(output)
```

Comment:
I understand the intention of these helpers, but it simply doesn't "feel right"(TM). First of all, there should be yardoc documentation here explaining what are they good for (mocking the dialog but still failing if the arguments are wrong). And even though... I don't like them. Can I finish my own HackWeek stuff before elaborating further?

Reply:
yes, no problem

Reply:
Too busy. Don't block the pull request because of me. :-)